services/api.ts

```diff
@@ -32,6 +32,18 @@ export const getAvailableBooks = async () => {
   return response
 }
 
+export const getFinishedBooks = async () => {
```

Comment:
🤔 did I suggest that your api functions be stored in 1 file? Apologies if I did - but it doesn't make sense and makes this file harder to manage

It would be great if the api functions are grouped together according to its resource / purpose.

E.g.


api/books.ts (CRUD book)
api/sessions.ts (login, register, authorize or whoami endpoint)
api/base.ts (where your `fetchApi` lives)